### PR TITLE
bpo-34336: Don't promote possibility to leave out typing.Optional

### PR DESCRIPTION
**Pull request by scop**
https://bugs.python.org/issue34336

### CODE REVIEW

File: Doc/library/typing.rst
Comment (gvanrossum) [inline Doc/library/typing.rst]: Please keep the first sentence (i.e. `Note that this is not the same concept as an optional argument, which is one that has a default.`). You might also add examples, e.g. `def (arg: int = 0)` vs. `def foo(a: Optional[int] = None)`.

**Comment by scop** [comment]
Ok, left that in and rephrased the rest of the paragraph to use the suggested examples.

**Review by gvanrossum — APPROVED** [review]
LGTM. I'm not sure if this deserves a news item -- I'd think not (it's just a minor documentation tweak) but another core dev can decide.

**Comment by ericvsmith** [comment]
I don't see any need for a news entry.

**Review by ilevkivskyi — APPROVED** [review]

**Comment by ilevkivskyi** [comment]
Hm, I also forgot to add backport labels. @scop will you have time to make manual backports to 3.6 and 3.7? Thanks!

(If no, then I can do this.)

**Comment by Mariatta** [comment]
@ilevkivskyi you can still add the backport labels after merge and see if it will backport cleanly.

**Comment by ilevkivskyi** [comment]
Oh, wow, I didn't know. Thank you Mariatta!

**Pull request merged**
